omniport/package.json

```diff
@@ -92,4 +93,4 @@
     "webpack-livereload-plugin": "^2.2.0",
     "webpack-merge": "^4.2.1"
   }
-}
+}
```

Comment:
Add a new line at the end of the file

Reply:
My linter automatically did this to me, Should I change it?

Reply:
yes i guess, its recommended to leave a new line at the end of the file